index.html

```diff
@@ -437,10 +437,7 @@ <h2 id="securing-with-jose">Securing JSON-LD Verifiable Credentials with JOSE</h
             <p>
                 The <code>typ</code> header parameter SHOULD be <code>vc+jwt</code>.
                 When present, the <code>cty</code> header parameter SHOULD be <code>vc</code>.
-                We consider use of the <code>cty</code> header parameter in this
-                way to be the means of differentiating this specification's use
-                of <code>vc+jwt</code> from others that wish to use an identical
-                media type.
+                The <code>cty</code> header parameter value can be used to differentiate between secured content of different types when using <code>vc+jwt</code>.
```

Comment:
I think this language needs a bit more work (to be applied to all occurences). I do not yet have a suggestion, but I expect to figure it out today.